assets/js/util.js

```diff
@@ -39,6 +40,17 @@ if (typeof Object.assign != 'function') {
 }
 /* eslint-enable */
 
+function debounce(func) {
```

Comment:
Wait, I think we lost our extra duration argument that used to be here to allow setting another duration?